manifest/crds/sparkoperator.k8s.io_scheduledsparkapplications.yaml

```diff
@@ -3686,8 +3713,6 @@ spec:
                     servicePort:
                       format: int32
                       type: integer
-                    serviceType:
```

Comment:
I just copied `charts/spark-operator-chart/crds/` in place of `manifest/crds/` without double checking
But it seems like that 2 PR were merged only applying changes to `manifest/crds/`  (#1240 and #1100)

I'll fix this right away

In the mean time, should we update the tooling to avoid those issue in the future ?

Reply:
Yes, looks like we need to update the tooling to make sure both places are updated consistently.